main.go

```diff
@@ -47,9 +47,10 @@ func init() {
 }
 
 func printVersion() {
-	log.Info(fmt.Sprintf("Operator Version: %s", version.Operator))
-	log.Info(fmt.Sprintf("Go Version: %s", runtime.Version()))
-	log.Info(fmt.Sprintf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH))
+	log.Info(fmt.Sprintf("version: v%s", version.VersionOperator))
+	log.Info(fmt.Sprintf("operator-sdk version: v%s", version.VersionSDK))
+	log.Info(fmt.Sprintf("go Version: %s", runtime.Version()))
+	log.Info(fmt.Sprintf("go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH))
```

Comment:
Why the downcasing?  The Go project name is canonically title case, e.g. see examples [here][1].

[1]: https://golang.org/doc/